api/apps/api/src/modules/scenarios/scenarios.controller.ts

```diff
@@ -64,6 +65,9 @@ import { ProxyService } from '@marxan-api/modules/proxy/proxy.service';
 const basePath = `${apiGlobalPrefixes.v1}/scenarios`;
 const solutionsSubPath = `:id/marxan/run/:runId/solutions`;
 
+const marxanRunTag = 'Marxan Run';
+const marxanFilesTag = 'Marxan Run - Files';
```

Comment:
especially if we want to expand the use of `@ApiTags()` as we go + retrospectively, maybe this tag may be called `marxanRunFilesTag` (we'd have `marxanFilesTags` for the range of file upload operations we support, for example)